configure.ac

```diff
@@ -353,6 +353,7 @@ AM_CONDITIONAL([OPENTHREAD_BUILD_TESTS], [test "${nl_cv_build_tests}" = "yes"])
 #  Full Thread Device
 #
 
+AC_MSG_CHECKING([Enable FTD])
```

Comment:
nit: looking at the other instances of `AC_MSG_CHECKING`, the wording seems to be more "colloquial".  For example:

```
checking for strip... strip
checking whether ln -s works... yes
checking for cmp... /usr/bin/cmp
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking whether gcc is Clang... yes
```

What about changing these message to be:

```
AC_MSG_CHECKING([whether to enable FTD])
```